src/Symfony/Component/Serializer/Encoder/XmlEncoder.php

```diff
@@ -498,4 +498,23 @@ private function createDomDocument(array $context): \DOMDocument
 
         return $document;
     }
+
+    /**
+     * @throws NotEncodableValueException
+     */
+    private function saveXml(\DOMDocument $DOMDocument, ?\DOMNode $node = null, ?int $options = null): string
+    {
+        $prevErrorHandler = set_error_handler(static function ($type, $message, $file, $line, $context = []) use (&$prevErrorHandler) {
+            if (\in_array($type, [\E_ERROR, \E_WARNING], true)) {
```

Comment:
```suggestion
            if ($type === \E_ERROR || $type === \E_WARNING) {
```
might be a bit faster

Reply:
Done